cmd/pkf/internal/http/handlers/pubkey.go

```diff
@@ -0,0 +1,174 @@
+package handlers
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/mailchain/mailchain/cmd/internal/datastore"
+	"github.com/mailchain/mailchain/cmd/internal/http/params"
+	"github.com/mailchain/mailchain/encoding"
+	"github.com/mailchain/mailchain/errs"
+	"github.com/mailchain/mailchain/internal/address"
+	"github.com/mailchain/mailchain/internal/pubkey"
+	"github.com/pkg/errors"
+)
+
+// GetPublicKey returns a handler get spec
+func GetPublicKey(store datastore.PublicKeyStore) func(w http.ResponseWriter, r *http.Request) {
+	// Get swagger:route GET /public-key PublicKey GetPublicKey
+	//
+	// Public key from address.
```

Comment:
Copy-pasted this from the other get public-key API. Need to make sure if the purpose of this endpoint is reflected correctly here